### DISCUSSION TITLE
Atom terminal cursor problem with shell mode

### DISCUSSION THREAD

**Discussion by bardo84** [atom/atom#22631] [Q&A]
Hi,

I am using Atom with the Juno Julia package.

After a shell command like ; ls
the input line looks like this:
cursor two blanks julia>
Functionality is there, just garbled (below from copy/paste)
1+1jul+12mjulia>
2

   julia> 

Any ideas?
Thx

**Comment by Dark403969** [comment]
> Hi,
> 
> I am using Atom with the Juno Julia package.
> 
> After a shell command like ; ls
> the input line looks like this:
> cursor two blanks julia>
> Functionality is there, just garbled (below from copy/paste)
> 1+1jul+12mjulia>
> 2
> 
> julia>
> 
> Any ideas?
> Thx

t
